_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,240 @@
+
+In `plugins` part, you just need to list all connectors, which you want ot have in your image. 
+Again, each item in the list has several options, which you need to specify like `name` of the plugin and `artifacts` section with info needed for download and work with it.
+Artifact `type` could be either `zip`, `tgz` or `jar` and of course has to be same as artifact available on `url`.
+`sha512sum` is optional property, which is used for validation of downloaded artifact by Strimzi, if it's specified.
+
+## Quick example
+
+Now lets go through quick example how to make it working in your cluster. 
+At first, you have to have Strimzi and Kafka cluster up and running. 
+Then you can create KafkaConnect with the following configuration (just change registry and organization in image name and secret):
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+  replicas: 1
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
+  config:
+    group.id: my-connect-cluster
+    offset.storage.topic: my-connect-cluster-offsets
+    config.storage.topic: my-connect-cluster-configs
+    status.storage.topic: my-connect-cluster-status
+    key.converter: org.apache.kafka.connect.json.JsonConverter
+    value.converter: org.apache.kafka.connect.json.JsonConverter
+    key.converter.schemas.enable: true
+    value.converter.schemas.enable: true
+    config.storage.replication.factor: 3
+    offset.storage.replication.factor: 3
+    status.storage.replication.factor: 3
+  build:
+    output:
+      type: docker
+      image: my-reg.io/my-org/my-connect-cluster:latest
+      pushSecret: my-registry-credentials
+    plugins: 
+      - name: debezium-postgres-connector
+        artifacts:
+          - type: zip
+            url: https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/1.4.2.Final/debezium-connector-postgres-1.4.2.Final-plugin.zip
+            sha512sum: ef1620547e6ddf5be010271849b6a87a19f6e6beee93b379c80883815b8f37ec5137095b2c99975d7704cbf957e6a33d76c61109582cad57c7cbbfae43adc86c
+      - name: camel-timer
+        artifacts:
+          - type: tgz
+            url: https://repo.maven.apache.org/maven2/org/apache/camel/kafkaconnector/camel-timer-kafka-connector/0.8.0/camel-timer-kafka-connector-0.8.0-package.tar.gz
+            sha512sum: c0102700ae176b1ef078bdb86d640935a1333865e58b870f6b0ef3d9dad067221122aee6bf52852dad209a7c2051f359e0434fc7fbd783fb50e21707c7577ef9
+      - name: echo-connector
+        artifacts:
+          - type: jar
+            url: https://github.com/scholzj/echo-sink/releases/download/1.1.0/echo-sink-1.1.0.jar
+            sha512sum: b7da48d5ecd1e4199886d169ced1bf702ffbdfd704d69e0da97e78ff63c1bcece2f59c2c6c751f9c20be73472b8cb6a31b6fd4f75558c1cb9d96daa9e9e603d2
```

Comment:
I think it is a bit weird to have basically the same example in this chapter as in the previous one. MAybe you can keep here only the Camel connector which is the only one you use? (you will also need to update the status below.)